gateway/1.4/modules/ROOT/pages/flex-conn-monitoring.adoc

```diff
@@ -0,0 +1,18 @@
+= Monitoring in Connected Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Connected Mode enables you monitor Flex Gateway APIs with Anypoint Platform. By using the Anypoint Platform, you can configure Flex Gateway logs and view Key Metrics and analytics.
+
+Get started monitoring Flex Gateway running in Connected Mode with the following tasks:
```

Comment:
This is hard to parse.

Use these tasks to get started with monitoring FG running in CM?

Reply:
Should these bullets below be imperative rather than gerunds?

Reply:
Did you disagree with my comment on line 9?